snooty/test_parser.py

```diff
@@ -615,3 +615,97 @@ def test_cond() -> None:
         </directive>
         </root>""",
     )
+
+
+def test_version() -> None:
```

Comment:
I'd actually name this `test_versionchange`

Reply:
See above, I named it `test_version` because it tests 3 different directives. But they are all functionally the same so I could see it making sense to use `test_versionchange` instead!